specs/openapi-spec.yaml

```diff
@@ -0,0 +1,322 @@
+openapi: 3.0.2
+info:
+  title: ION Storage
+  version: '${project.version}'
+  description: ION storage abstraction service
+  contact:
+    name: 'Connexta, LLC'
+    url: 'https://connexta.com'
+  license:
+    name: GNU Lesser General Public License
+    url: 'http://www.gnu.org/licenses/lgpl.html'
+servers:
+- url: '${rest.server.url}'
+
+paths:
+  /folders:
```

Comment:
✏️ This doesn't work like a path to a folder on a file system, right. I think "bucket" is more appropriate.